### ISSUE TITLE
iptables was not "restarting" as it was not installed

### ISSUE THREAD

**Issue by z3cka** [UCLALibrary/ansible_uclalib_role_iptables#8]
```
NOTIFIED: [ansible_uclalib_role_iptables | restart iptables] ******************
<rancher.ucla.sey.gr> ESTABLISH CONNECTION FOR USER: root on PORT 65492 TO rancher.ucla.sey.gr
<rancher.ucla.sey.gr> REMOTE_MODULE service name=iptables state=restarted
<rancher.ucla.sey.gr> EXEC /bin/sh -c 'mkdir -p $HOME/.ansible/tmp/ansible-tmp-1474996132.09-268370433248322 && echo $HOME/.ansible/tmp/ansible-tmp-1474996132.09-268370433248322'
<rancher.ucla.sey.gr> PUT /var/folders/m9/xsc48f0d6ls1rynfdw179p2mj7q5hj/T/tmpVvMbZP TO /root/.ansible/tmp/ansible-tmp-1474996132.09-268370433248322/service
<rancher.ucla.sey.gr> EXEC /bin/sh -c 'LANG=en_US.UTF-8 LC_CTYPE=en_US.UTF-8 /usr/bin/python /root/.ansible/tmp/ansible-tmp-1474996132.09-268370433248322/service; rm -rf /root/.ansible/tmp/ansible-tmp-1474996132.09-268370433248322/ >/dev/null 2>&1'
failed: [rancher.ucla.sey.gr] => {"failed": true}
msg: FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `mangle': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `nat': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `filter': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `mangle': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `mangle': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `mangle': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `nat': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `nat': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `nat': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `filter': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `filter': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables v1.4.7: can't initialize iptables table `filter': Permission denied (you must be root)
Perhaps iptables or your kernel needs to be upgraded.
FATAL: Could not load /lib/modules/3.16.0-4-amd64/modules.dep: No such file or directory
iptables-restore v1.4.7: iptables-restore: unable to initialize table 'filter'

Error occurred at line: 1
Try `iptables-restore -h' or 'iptables-restore --help' for more information.


FATAL: all hosts have already failed -- aborting

NOTIFIED: [ansible_uclalib_role_iptables | restart iptables] ******************
FATAL: no hosts matched or all hosts have already failed -- aborting


FATAL: all hosts have already failed -- aborting

NOTIFIED: [ansible_uclalib_role_iptables | restart iptables] ******************
FATAL: no hosts matched or all hosts have already failed -- aborting


FATAL: all hosts have already failed -- aborting

PLAY RECAP ********************************************************************
           to retry, use: --limit @/Users/cgrzecka/uclalib_solr4local_rhel.retry

rancher.ucla.sey.gr        : ok=44   changed=27   unreachable=0    failed=1
```

PR coming

**Comment by z3cka** [comment]
#9 fixes this.

**Comment by z3cka** [comment]
fixed by #9... @sgurnick I wonder if it had to do with the version of ansible I'm running— which is 1.9.2
